lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
+    process.stdout._handle.setBlocking(true)
+    process.stderr._handle.setBlocking(true)
+  }
```

Comment:
Actually, i'm curious, does this actually work when done retroactively?

If you have async writes and they you do this does it immediately cause a blocking flush on call? If you don't know, that's fine, I'll investigate.